### PR TITLE
State query cleanup

### PR DESCRIPTION
**Pull request by jhford**
I still need to test this in staging, but this change removes a bunch of complexity as well as fixing what could've been a small, edge-case bug in how state is tracked.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -161,6 +161,10 @@ let load = loader({
         }
       }
 
+      if (process.env.FORCE_JS_PG_CLIENT) {
```
Comment (imbstack) [inline lib/main.js]: Why not make this a config.yml thing?

File: lib/cloud-watch-event-listener.js [hunk 2]
```diff
@@ -147,39 +142,33 @@ class CloudWatchEventListener extends events.EventEmitter {
       } 
 
       if (dbResponse.lastEvent && dbResponse.lastEvent < generated) {
-        try {
```
Comment (imbstack) [inline lib/cloud-watch-event-listener.js]: I'm confused why we can get rid of these try-catch now.

File: lib/main.js (hunk 1 above)
Comment (jhford) [reply lib/main.js]: it's only useful to get better errors when a SQL query fails to parse, only really useful when doing development.  I'd rather not bake that into the config.yml, since we really should be doing all testing, everwhere with PG-Native.  Doing this is explicitly an attempt to make this environment variable non-discoverable.

I'd rather delete this from the code altogether than put it in config.yml

File: lib/cloud-watch-event-listener.js (hunk 2 above)
Comment (jhford) [reply lib/cloud-watch-event-listener.js]: Because I missed it while doing my pass through.  Thanks for catching it.

Comment (jhford) [reply lib/cloud-watch-event-listener.js]: Hmm, thinking about it more, we can probably get rid of the dbResponse.lastEvent checking here altogether and rely on the upsert's `ON CONFLICT... WHERE instances.terminated < $9` thingy.

**Comment by jhford** [comment]
@imbstack hey, thanks for the catch on the cloud watch event listener.  I've drastically simplified the function based on your spotting that, thanks!
